extras/ioc_hash_scan.sh

```diff
@@ -0,0 +1,160 @@
+#!/bin/bash
+#
+# This script will scan AWS instances occurances of IOC hashes listed in the blob at src/ioc_scan/ioc_scanner.py
```

Comment:
Fix typo and rewrap comment:
```suggestion
# This script will scan AWS instances for occurrences of IOC hashes listed in
# the blob at src/ioc_scan/ioc_scanner.py
```

Reply:
@bra1ncramp - Where are you testing this?  Can you try changing
```hcl
  # Allow the user to terminate his or her own sessions
  statement {
    actions = [
      "ssm:TerminateSession",
    ]
    resources = [
      "arn:aws:ssm:${var.aws_region}:${local.this_account_id}:session/${aws:username}-*",
    ]
  }
```
to
```hcl
  # Allow the user to terminate his or her own sessions
  statement {
    actions = [
      "ssm:TerminateSession",
    ]
    resources = [
      "arn:aws:ssm:${var.aws_region}:${local.this_account_id}:session/${aws:username}-*",
    ]
  }
```
in the policy `StartStopSSMSession`?  (Note that the only change is that the ampersand (`&`) is changed to a dollar sign (`$`) toward the end of the resource ARN.)  You can simply make this change live, in the AWS console for now.